app/services/pledge_form_generator.rb

```diff
@@ -0,0 +1,144 @@
+class PledgeFormGenerator
+  attr_reader :patient, :user, :case_manager_name
+
+  def initialize(user, patient, case_manager_name)
+    @patient = patient
+    @user = user
+    @case_manager_name = case_manager_name.to_s
+  end
+
+  def patient_amount
+    @patient.pregnancy.dcaf_soft_pledge
```

Comment:
@colinxfleming wondering if the case_manager_name should get to_s'd at a higher level and we just receive a string of the name?